mongo/tests/mocked_api.py

```diff
@@ -35,9 +35,17 @@ def __init__(self, db_name, coll_name):
             with open(os.path.join(HERE, "fixtures", "system.replset"), 'r') as f:
                 content = json.load(f, object_hook=json_util.object_hook)
                 self.find_one = MagicMock(return_value=content)
-        else:
-            with open(os.path.join(HERE, "fixtures", f"indexStats-{coll_name}"), 'r') as f:
-                self.aggregate = MagicMock(return_value=json.load(f, object_hook=json_util.object_hook))
+        # else:
+        #     with open(os.path.join(HERE, "fixtures", f"indexStats-{coll_name}"), 'r') as f:
+        #         self.aggregate = MagicMock(return_value=json.load(f, object_hook=json_util.object_hook))
```

Comment:
Why is this left out? There is no `else` statement now

Reply:
the else is moved the mocked `aggregate` method. if any method that falls out of the mocking, we expect it to fail